tests/modules/programs/mpv/mpv-stubs.nix

```diff
@@ -1,26 +1,27 @@
-{ pkgs, ... }:
-
 {
   nixpkgs.overlays = [
-    (self: super: {
-      mpvScript = pkgs.runCommandLocal "mpvScript" { scriptName = "something"; }
+    (final: prev: {
+      mpvScript = prev.runCommandLocal "mpvScript" { scriptName = "something"; }
         "mkdir $out";
 
-      mpv-unwrapped = super.mpv-unwrapped.overrideAttrs {
-        builder = pkgs.writeShellScript "dummy" ''
-          PATH=${pkgs.coreutils}/bin
+      mpv-unwrapped = let
+        lua = prev.emptyDirectory.overrideAttrs {
+          luaversion = "0";
+          passthru.withPackages = pkgsFn: prev.emptyDirectory;
```

Comment:
I put `withPackages` inside `passthru` to avoid another evaluation failure.

```
           19|         luaversion = "0";
           20|         withPackages = pkgsFn: prev.emptyDirectory;
             |         ^
           21|       };

       error: cannot coerce a function to a string
```